metanovaverse/assetlist.json

```diff
@@ -0,0 +1,32 @@
+{
+  "$schema": "../assetlist.schema.json",
+  "chain_name": "metanovaverse",
+  "assets": [
+    {
+      "description": "MetaNova Verse Native Token",
+      "denom_units": [
+        {
+          "denom": "amnv",
+          "exponent": 0
+        },
+        {
+          "denom": "mnv",
+          "exponent": 18
+        }
+      ],
+      "base": "amnv",
+      "name": "MNV",
+      "display": "mnv",
+      "symbol": "MNV",
+      "type_asset": "sdk.coin",
+      "images": [
+        {
+          "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/blockx/images/mnova.png"
```

Comment:
The image is being uploaded under /metamovaverse/images/, not /blockx/images/

Reply:
Also, can you please upload a square image, ideally with a transparency layer where the whitespace is, instead of the color white, and no extra padding